docs/reference/schema.md

```diff
@@ -638,6 +638,147 @@ addtargets:
 ---
 ```
 
+### Vulnerability
```

Comment:
No 'This code is referened by' sections here?

Reply:
See https://github.com/GFDRR/rdl-standard/pull/124#discussion_r1255006311